internal/yggdrasil/yggdrasil.go

```diff
@@ -508,6 +515,33 @@ func (h *Handler) toWorkloadList(ctx context.Context, logger logr.Logger, deploy
 	return list, nil
 }
 
+// validatePodmanLabels validate the podman labels
+func validatePodmanLabels(podmanLabels map[string]string, deployment v1alpha1.EdgeDeployment) bool {
+	for key, value := range podmanLabels {
+		if strings.Contains(key, "io.containers.autoupdate.authfile") {
+			for _, ctr := range deployment.Spec.Pod.Spec.Containers {
+				for _, vm := range ctr.VolumeMounts {
+					if value == vm.MountPath {
+						return true
```

Comment:
if there are 2 labels and an authfile was provided only for the first one, this function returns true while it should fail.

Reply:
You mean for example:
```yaml
labels:
  podman/io.containers.autoupdate.authfile: /tmp/auth
  labelX: value1
  labelY: value2

containers:
  ...
  mount:
    path: /tmp/auth
```

In such case we return true. What use case do you have exactly in mind?

Reply:
My bad :-)
I mistakenly assumed that more than one authfile can be provided.